apps/docs/content/3-components/2-library/link/design.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Link'
+description: 'Link Design Guidelines'
+status: 'stable'
+component:
+  id: 'link'
+  link: 'https://www.figma.com/design/ZEqfpXhocdZs1K5YfsE5ws/Design-system?node-id=4359-2198'
+  status: 'stable'
+---
+
+# Link
+
+Links are blue and underlined by default. If your link is at the end of a sentence or paragraph, make sure that the linked text does not include the full stop.
+
+<Section current="design">
+
+<ComponentPreview>
+  <Link href="#">Link text</Link>
+</ComponentPreview>
+<FigmaPreviewButton href="https://www.figma.com/design/ZEqfpXhocdZs1K5YfsE5ws/Design-system?node-id=4359-2198" />
+
+## States
+
+<ComponentPreviewLayout className="flex-col">
+
+<ComponentPreviewItem className="w-105">
+  <Link href="#" external>Link text</Link>
+</ComponentPreviewItem>
+
+<ComponentPreviewItem className="w-105">
+  <Link disabled href="#" external>Link Disabled</Link>
+</ComponentPreviewItem>
+
+</ComponentPreviewLayout>
+
+## Variants
+
+<ComponentPreviewLayout className="flex-col">
+
+
+<ComponentPreviewItem className="w-105">
+  <Link asButton={{ appearance: 'default', size: 'medium', variant: 'primary'}} href="#"> Link text</Link>
+</ComponentPreviewItem>
+<ComponentPreviewItem className="w-105">
+  <div className="gi-bg-black gi-p-4 gi-w-fit">
+    <Link appearance="light" href="/light" external>Link text</Link>
+  </div>
+</ComponentPreviewItem>
+<ComponentPreviewItem className="w-105">
+  <Link href="#" iconEnd="arrow_forward" iconStart="arrow_back" external>Link no underline</Link>
+</ComponentPreviewItem>
+
+</ComponentPreviewLayout>
+
+## Usage
+
+Use the Link component to create consistent, accessible, and visually appealing hyperlinks. It handles external links, disabled states, icons, and different visual styles like underline and color inheritance.
+
+### Do
+
+- Use for simple navigational hyperlinks (internal or external).
+- Use **external** to properly mark and secure external links (adds rel="noopener noreferrer" and target="_blank").
+- Use **iconStart** and **iconEnd** to enhance clarity with visual indicators.
+- Use **noUnderline**, **noVisited**, or **noColor** to match custom style requirements.
+- Use **asChild** to wrap custom components or slots while preserving semantics.
+
+### Don't
+
+- Don't place block-level elements like ```<div>``` or ```<p>``` directly inside the Link — wrap content in ```<span>``` or use asChild.
+- Don't use this for SPA routing — prefer ```<Link>``` from Next.js or React Router.
```

Comment:
This is incorrect advice. We have added a feature to support external Link components using the `asChild` property.
So Next.js or React Router links need to be wrapped with our Link component.